pubs/plugs/git/git.py

```diff
@@ -0,0 +1,94 @@
+
+import subprocess
+from pipes import quote as shell_quote
+
+from ...plugins import PapersPlugin
+from ...events import *
+
+
+class GitPlugin(PapersPlugin):
+
+    name = 'git'
+    pubsdir = None
+
+    def __init__(self, conf):
+        self.description = "Run git commands in the pubs directory"
+        # Needed for the event listening
+        GitPlugin.pubsdir = conf['main']['pubsdir']
```

Comment:
`self.pubsdir`: we want it to be instance specific (although in practice we use the class as a singleton, there is no need to assume that in the class design).

Reply:
OH, i just realized looked into how get_instance works. That makes a lot more sense now. Thats a really smart way of getting saved instances.